site/content/en/docs/user/stages-configuration.md

```diff
@@ -74,6 +74,60 @@ and gain insights into the behavior and performance of your applications and inf
 
 The `<expressions-string>` is provided by the [Go Implementation] of [JQ Expressions]
 
+## How `kwok` applies Stages
+
+Stages can be generally divided into two categories based on different settings of the `next` field.
+A Stage that has a non-empty `nextTemplate` is a "Change Stage". The content of `nextTemplate` will be used by `kwok` for updating resource status.
```

Comment:
```suggestion
A Stage that has a non-empty `statusTemplate` is a "Change Stage", it will be used for updating resource status.
```

Reply:
This causes a _comma splice_ grammar error if using a comma to join two independent clauses. I will change it to "which".